RELEASE-NOTES.txt

```diff
@@ -10,6 +9,10 @@
 * [**] [Jetpack-only] Help: Display the Jetpack app FAQ card on Help screen when switching from the WordPress app to the Jetpack app is complete. [#20232]
 * [***] [Jetpack-only] Blaze: We added support for Blaze in the app. The user can now promote a post or page from the app to reach new audiences. [#20253]
 
+21.8.1
+-----
+* [**] [internal] Fixes a crash that happens in the background when the weekly roundup notification is being processed. [#20275]
```

Comment:
I realize we don't often have raw release notes for hotfixes, but since this was added in the original PR, I decided to keep it and make it explicit in which release it was introduced, even though if one looked at the Git log once the process is finished, it would be part of the changes coming from 21.9.

Also worth noting that this will not be part of the release notes for 21.8.1. For hotfixes, we use the same notes as the base release.